internal/sql/driver/driver_test.go

```diff
@@ -53,6 +57,16 @@ func TestParseDSN(t *testing.T) {
 				},
 			},
 		},
+		{
+			DSN:           "",
+			Serialization: &serialization.Config{PortableVersion: 2},
+			Pre: func() {
+				idriver.SetSerializationConfig(&serialization.Config{PortableVersion: 2})
+			},
+			Post: func() {
+				idriver.SetSerializationConfig(nil)
+			},
+		},
```

Comment:
We don't need to test others right? (Logger and SSL config setter)

Reply:
I've added them at: https://github.com/hazelcast/hazelcast-go-client/pull/728/commits/858ee6604d144eb445053d128ef8764ce720e156